private_gpt/ui/ui.py

```diff
@@ -168,6 +170,12 @@ def build_history() -> list[ChatMessage]:
                     role=MessageRole.SYSTEM,
                 ),
             )
+        def draw_methods(service_type):
+            service = getattr(self, f'_{service_type}_service')
+            return {
+                True: getattr(service, f'stream_{service_type}'),
+                False: getattr(service, f'{service_type}')
+            }
```

Comment:
I don't think reflection is necessary in this case. It complicates the understanding of the method a lot.

Reply:
Consider use lambda function or conventional functions and a switch statement

Reply:
My apologies on merging main after the commit